pages/common/vimdiff.md

```diff
@@ -0,0 +1,24 @@
+# vimdiff
+
+> Open up to four files in Vim and show their differences.
+> See Vim documentation for how to edit and navigate within a window.
+
+- Open files and show the differences:
+
+`vimdiff {{file1}} {{file2}}`
+
+- Use horizontal window split instead of the default vertical split:
+
+`vimdiff -o {{file1}} {{file2}}`
+
+- Close current window without saving changes:
```

Comment:
oh cool. I didn't know that and didn't notice the backticks. I've removed the `:q!` and `:w` and made the `Ctrl + w` consistent with other's documentation. Thanks :smile: